### ISSUE TITLE
missing README despite repo having one

### ISSUE THREAD

**Issue by noformnocontent** [npm/www#161]
npm: https://www.npmjs.com/package/passport
repo: https://github.com/jaredhanson/passport

![screenshot from 2017-04-07 16-07-58](https://cloud.githubusercontent.com/assets/23745012/24803764/8b5b5a68-1bac-11e7-91e8-47baa1c40c19.png)

**Comment by aearly** [comment]
This may be a clue:  the `.npmignore` for the project lists the readme, but npm never ignores a readme during publish.  Perhaps there is a discrepancy in the logic for adding the readme to the website.

**Comment by aearly** [comment]
The published tarball doesn't have a `README.md`: https://registry.npmjs.org/passport/-/passport-0.3.2.tgz

This is strange, the `npm` CLI shouldn't ignore a README.

**Comment by damoclark** [comment]
I have experienced the very same issue.  

I just uploaded a new package to the registry [node-persistent-queue](https://www.npmjs.com/package/node-persistent-queue) and it isn't displaying the README.md either.  I checked the [published tarball](https://registry.npmjs.org/node-persistent-queue/-/node-persistent-queue-0.1.0.tgz) and it does include the README.md file so it was uploaded by the `npm` cli.

I do not have an `.npmignore` in my project.
